src/org/zaproxy/zap/extension/jxbrowserlinux64/selenium/JxBrowserProvider.java

```diff
@@ -99,8 +99,8 @@ private RemoteWebDriver getRemoteWebDriver(final String proxyAddress, final int
 
     private RemoteWebDriver getRemoteWebDriverImpl(String proxyAddress, int proxyPort) {
         try {
-            if (zbf == null) {
-                zbf = new ZapBrowserFrame(false, true, false, false);
+            if (zbf == null || zbf.isClosed()) {
+                zbf = new ZapBrowserFrame(true, true, false, true);
```

Comment:
We should check the `requesterId` to not add the tool bar and new tab button when launching browsers for AJAX Spider.